python/geom.py

```diff
@@ -234,6 +234,27 @@ def transform(self, m):
 
         for s in self.H_susceptibilities:
             s.transform(m)
+    def get_eps(self,freq):
+
+        # Clean the input
+        freq = np.squeeze([freq])
+
+        # Compensate for scalar inputs
+        if len(freq.shape) == 0:
+            freq = np.array([freq])
+
+        # Initialize with instantaneous dielectric tensor, use numpy arrays for
+        # convenience and speed.
+        eps = np.array([self.epsilon_diag.x,self.epsilon_diag.y,self.epsilon_diag.z],dtype='complex128')
```

Comment:
What about the off-diagonal parts?

Reply:
Just fixed this by returning the entire dielectric tensor as a list. 

Do you know of any good materials to test this with? The uniaxial materials below obviously only test different values along the diagonal, but not any off-diagonal elements. I'm having trouble finding dispersion data for the off-diagonal terms of triclinic/monoclinic materials.